test/hotspot/jtreg/compiler/jvmci/compilerToVM/IterateFramesNative.java

```diff
@@ -48,19 +44,21 @@
  *                   -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCI
  *                   -XX:+DoEscapeAnalysis -XX:-UseCounterDecay
  *                   compiler.jvmci.compilerToVM.IterateFramesNative
+ * @run main/othervm -Xbatch -Xcomp -Xbootclasspath/a:.
```

Comment:
I think `-Xcomp` implies `-Xbatch`.
I think you should add `-XX:CompileOnly=jdk.vm.ci.hotspot.CompilerToVM::iterateFrames` since you only really care about that method being compiled for sake of the test.

Reply:
Hm, actually, I don't really need to use `-Xcomp` since the test should already trigger compilation anyway. It was just a way to ensure it's really compiled. I'll replace it with an assert in the test that the method is compiled.

Reply:
Reverted to using `-Xcomp` but with `-XX:CompileOnly`. Some configurations did not trigger compilation in the expected iteration; I've probably just got the threshold wrong, but I think the `-Xcomp` solution is less fragile and thus preferable.